src/content/docs/es/tutorial/6-islands/1.mdx

```diff
@@ -57,7 +57,7 @@ Este componente tomará un array de mensajes de bienvenida como props y seleccio
 
       return (
         <div> 
-          <h3>{greeting}!Gracias por su visita¡.</h3>
+          <h3>{greeting} ¡Gracias por su visita!</h3>
```

Comment:
```suggestion
          <h3>{greeting}¡Gracias por su visita!</h3>
```

Reply:
I have added this space because otherwise the texts would be joined together.:

"Hola¡Gracias por su visita!" --> "Hola ¡Gracias por su visita!" 

I find it strange that there is no space to separate the words in any of the languages. But those are silly details hehe

Reply:
Woops, you're right.